frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -278,7 +277,10 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
         const scList = _.filter(storageClasses, (sc) => sc.provisioner === provisioner);
         // take the default storageclass
         _.forEach(scList, (sc) => {
-          if (sc.metadata && _.isEqual(sc.metadata.annotations, defaultSAnotations)) {
+          if (
+            sc.metadata &&
+            _.isEqual(sc.metadata.annotations, { [defaultClassAnnotation]: 'true' })
```

Comment:
You should only check the value of the single annotation. Otherwise another annotation on the storage class would cause the check to fail. But better to use the existing `isDefaultClass` helper from storage.class.tsx.